core/src/main/se/tevej/game/controller/screen/DigitScreen.java

```diff
@@ -0,0 +1,18 @@
+package main.se.tevej.game.controller.screen;
+
+public abstract class DigitScreen {
+
+    @SuppressWarnings("PMD.DefaultPackage")
+    /* package */ final ChangeScreen screenChanger;
+
+    /* package */ DigitScreen(ChangeScreen screenChanger) {
+        this.screenChanger = screenChanger;
+    }
```

Comment:
add a comment or "justification" to clarify why you supress these warnings

Reply:
cause checkstyle

Reply:
not good enough :<

Reply:
i fixed a comment

Reply:
yes, now fix a good one